cli/valet.php

```diff
@@ -533,6 +534,11 @@
             $site = basename(getcwd());
         }
 
+        if (! $phpVersion) {
```

Comment:
Could we save this change for a separate PR? I think you're already building it in #1213, right?